base_layer/wallet/src/output_manager_service/service.rs

```diff
@@ -848,13 +896,13 @@ where
             unique_id,
             fee_per_gram,
         );
-        let output_features = OutputFeatures::default();
+        let output_features_estimate = OutputFeatures { ..Default::default() };
```

Comment:
This is the same thing

Reply:
The wording choice of `output_features_estimate` is important here to let the reader know at this stage these features are not the real features and only used to for fee estimation.

Reply:
Think he means `OutputFeatures { ..Default::default() };` is a convoluted way of writing the previous code `OutputFeatures::default()`

Reply:
Yip, that was a refactoring remnant.